package.json

```diff
@@ -14,11 +14,11 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "amoeba": "0.4.1",
-    "lodash": "3.4.0"
+    "amoeba": "0.6.0",
+    "lodash": "4.17.11"
```

Comment:
I reckon that you can ditch `lodash` altogether. The only place I see it referenced is https://github.com/tidepool-org/gatekeeper-client/blob/master/lib/authorizationClient.js#L18, but then `_` isn't even used in the rest of the file...